mobile/src/main/res/xml/preferences.xml

```diff
@@ -31,6 +31,10 @@
             android:clickable="true"
             android:key="default_openhab_clear_default_sitemap"
             android:title="@string/settings_clear_default_sitemap" />
+        <SwitchPreference
```

Comment:
I wonder whether we shouldn't introduce a new category here, since both sitemap prefs aren't connection related...can't think of a good header string at the moment though :-(